apps/fuzzyclock/fuzzy_clock.star

```diff
@@ -62,120 +59,192 @@ numbersPerLang = {
         10: "TEN",
         11: "ELEVEN",
         12: "TWELVE",
-        15: "QUARTER",
-        20: "TWENTY",
-        25: "TWENTY-FIVE",
-        30: "HALF",
     },
 }
 numbersPerLang["en-GB"] = numbersPerLang["en-US"]
 numbersPerLang["nl-BE"] = numbersPerLang["nl-NL"]
+numbersPerLang["de-AT"] = numbersPerLang["de-DE"]
+numbersPerLang["de-DE-alt"] = numbersPerLang["de-DE"]
+numbersPerLang["de-DE-alt2"] = numbersPerLang["de-DE"]
 numbersPerLang["de-CH"] = numbersPerLang["de-DE"]
+numbersPerLang["de-CH-alt"] = numbersPerLang["de-DE"]
 
-wordsPerLang = {
-    "nl-NL": {
-        "hour": "UUR",
-        "half": "HALF",
-        "to": "VOOR",
-        "past": "OVER",
+patternsPerLang = {
+    "de-AT": {
+        0: "{hour}\nUHR",
+        5: "FÜNF\nÜBER\n{hour}",
+        10: "ZEHN\nÜBER\n{hour}",
+        15: "VIERTEL\nÜBER\n{hour}",
+        20: "ZEHN\nVOR HALB\n{next_hour}",
+        25: "FÜNF\nVOR HALB\n{next_hour}",
+        30: "HALB\n{next_hour}",
+        35: "FÜNF\nNACH HALB\n{next_hour}",
+        40: "ZEHN\nNACH HALB\n{next_hour}",
+        45: "VIERTEL\nVOR\n{next_hour}",
+        50: "ZEHN\nVOR\n{next_hour}",
+        55: "FÜNF\nVOR\n{next_hour}",
```

Comment:
For readability sake I'd prefer if we did something like `"FÜNF,VOR,{next_hour}"` (csv style), and then change L247 to `return pattern.format(hour = cur_hour, next_hour = next_hour).split(",")`

Reply:
Done